### ISSUE TITLE
[JENKINS-59602] Localizations of the warnings next generation plugin

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/warnings-ng-plugin#2909]
<p>All UI labels and help texts for the warnings plugin are internationalized already. <br/>
What is missing is the localization (l10n).</p>

<p>The old plugins (warnings, findbugs, checkstyle, analysis-core) where already localized, maybe it helps to copy some of these files.</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/drulli.png" title="drulli's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli">drulli</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602" target="_blank">Localizations of the warnings next generation plugin</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: warnings-ng-plugin
<li><b>label(s)</b>: help-wanted, newbie-friendly
<li><b>resolution</b>: Postponed
<li><b>resolved</b>: 2024-08-21T15:49:49+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 4
<li><b>imported</b>: 2025-12-09
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>All UI labels and help texts for the warnings plugin are internationalized already. 
What is missing is the localization (l10n).</p>

<p>The old plugins (warnings, findbugs, checkstyle, analysis-core) where already localized, maybe it helps to copy some of these files.</p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>udit_damare:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433247&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433247">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hello <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a>, Martin(@mPokornyETM) asked me to look in this issue. This is my first issue and can you help me with getting started on this?
I am currently googling localization and how its done. 
Also I need to find  where is the github repo to get started.</p>



<p>I will ask more specific doubts once I get started</p>
</pre>
</details>

</ul>
</details>

<p>Hello <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a>, Martin(@​mPokornyETM) asked me to look in this issue. This is my first issue and can you help me with getting started on this?<br/>
I am currently googling localization and how its done. <br/>
Also I need to find  where is the github repo to get started.</p>



<p>I will ask more specific doubts once I get started</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>udit_damare:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433248&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433248">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Can you share me github plugin repo link? <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a> 
Is it this one? 
gitlab-plugin</p>

<p><a href="https://github.com/Subzer0570/gitlab-plugin" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/Subzer0570/gitlab-plugin</a></p>
</pre>
</details>

</ul>
</details>

<p>Can you share me github plugin repo link? <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a> <br/>
Is it this one? <br/>
gitlab-plugin</p>

<p><a href="https://github.com/Subzer0570/gitlab-plugin" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/Subzer0570/gitlab-plugin</a></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/drulli.png" title="drulli's avatar" /> drulli:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433251&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433251">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>No, here is the link: <a href="https://github.com/jenkinsci/warnings-ng-plugin" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/warnings-ng-plugin</a></p>

<p>Before starting with translations it would make sense to check if we can use <a href="https://crowdin.com" class="external-link" target="_blank" rel="nofollow noopener">https://crowdin.com</a> here as well. See also: <a href="https://www.jenkins.io/doc/developer/crowdin/" class="external-link" target="_blank" rel="nofollow noopener">https://www.jenkins.io/doc/developer/crowdin/</a></p>
</pre>
</details>

</ul>
</details>

<p>No, here is the link: <a href="https://github.com/jenkinsci/warnings-ng-plugin" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/warnings-ng-plugin</a></p>

<p>Before starting with translations it would make sense to check if we can use <a href="https://crowdin.com" class="external-link" target="_blank" rel="nofollow noopener">https://crowdin.com</a> here as well. See also: <a href="https://www.jenkins.io/doc/developer/crowdin/" class="external-link" target="_blank" rel="nofollow noopener">https://www.jenkins.io/doc/developer/crowdin/</a></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>udit_damare:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433317&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433317">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Okay sir, I'll check on crowdin</p>
</pre>
</details>

</ul>
</details>

<p>Okay sir, I'll check on crowdin</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>udit_damare:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433376&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433376">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hello <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a> ,Can you please explain me the task?
Do I need to know any native language to work on this ?
What part  of the code I'll be working on?</p>

<p>Is there any time limit? I am a working SDE, I am trying to give atleast 2-3 hrs daily, Will this be enough time ?
<a href="https://www.jenkins.io/doc/developer/crowdin/" class="external-link" target="_blank" rel="nofollow noopener">https://www.jenkins.io/doc/developer/crowdin/</a>  Will I doing the whole setup or a part is done?</p>
</pre>
</details>

</ul>
</details>

<p>Hello <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a> ,Can you please explain me the task?<br/>
Do I need to know any native language to work on this ?<br/>
What part  of the code I'll be working on?</p>

<p>Is there any time limit? I am a working SDE, I am trying to give atleast 2-3 hrs daily, Will this be enough time ?<br/>
<a href="https://www.jenkins.io/doc/developer/crowdin/" class="external-link" target="_blank" rel="nofollow noopener">https://www.jenkins.io/doc/developer/crowdin/</a>  Will I doing the whole setup or a part is done?</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/drulli.png" title="drulli's avatar" /> drulli:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433399&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433399">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Actually there are two (or three tasks). </p>

<p>1. Enable crowdin for the warnings plugin.
2. Add some translations (what language are you speaking?).
3. Fix the texts that do not yet support i18n.</p>

<p>There is no time limit. So you basically start with step 1 and the work time boxed. When you have done enough to understand the code basis you can simply jump to a different topic (e.g., code), even in a different plugin. I think it is just good to see when someone makes the first step.</p>

<p>This issue is there for an ongoing support. If you want to have it split into several subtasks please let me know. </p>
</pre>
</details>

</ul>
</details>

<p>Actually there are two (or three tasks). </p>

<p>1. Enable crowdin for the warnings plugin.<br/>
2. Add some translations (what language are you speaking?).<br/>
3. Fix the texts that do not yet support i18n.</p>

<p>There is no time limit. So you basically start with step 1 and the work time boxed. When you have done enough to understand the code basis you can simply jump to a different topic (e.g., code), even in a different plugin. I think it is just good to see when someone makes the first step.</p>

<p>This issue is there for an ongoing support. If you want to have it split into several subtasks please let me know. </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>udit_damare:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433409&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433409">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I am from India and I speak hindi. 
I think once I have progressed considerably in step 1. I will get to know what I can do in 2nd task.
And about sub tasks, I'll let you know
Edit: by step 1 I meant R&amp;D. I apologize if I meant setting up crowdin</p>
</pre>
</details>

</ul>
</details>

<p>I am from India and I speak hindi. <br/>
I think once I have progressed considerably in step 1. I will get to know what I can do in 2nd task.<br/>
And about sub tasks, I'll let you know<br/>
Edit: by step 1 I meant R&D. I apologize if I meant setting up crowdin</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>udit_damare:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-59602?focusedId=433604&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-433604">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hello <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a>, I am stuck on setting up crowdin, I asked on gitter(gsoc-sig) and crowdin setup should be done by maintainer. Can you check my messages and tell me what can I do?
My username @Subzer0570</p>
</pre>
</details>

</ul>
</details>

<p>Hello <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=drulli" class="user-hover" rel="drulli">drulli</a>, I am stuck on setting up crowdin, I asked on gitter(gsoc-sig) and crowdin setup should be done by maintainer. Can you check my messages and tell me what can I do?<br/>
My username @​Subzer0570</p>
